server/routes/activitiesMigration/activitiesMigrationRouter.ts

```diff
@@ -6,46 +7,45 @@ import NomisMigrationService from '../../services/nomisMigrationService'
 import NomisPrisonerService from '../../services/nomisPrisonerService'
 import ActivitiesService from '../../services/activitiesService'
 import ActivitiesNomisMigrationService from '../../services/activities/activitiesNomisMigrationService'
+import authorisationMiddleware from '../../middleware/authorisationMiddleware'
+import { MIGRATE_ACTIVITIES_ROLE, MIGRATE_NOMIS_SYSCON } from '../../authentication/roles'
 
-export interface Services {
+export default function routes({
+  activitiesNomisMigrationService,
+  nomisMigrationService,
+  nomisPrisonerService,
+  activitiesService,
+}: {
   activitiesNomisMigrationService: ActivitiesNomisMigrationService
   nomisMigrationService: NomisMigrationService
   nomisPrisonerService: NomisPrisonerService
   activitiesService: ActivitiesService
-}
-export default function routes(router: Router, services: Services): Router {
+}): Router {
+  const router = express.Router({ mergeParams: true })
+  router.use(authorisationMiddleware([MIGRATE_ACTIVITIES_ROLE, MIGRATE_NOMIS_SYSCON]))
```

Comment:
each router now has the authorisation middleware which ensures that all the pages in the router are protected